src/Magento/FunctionalTestingFramework/Config/FileResolver/Root.php

```diff
@@ -28,7 +29,7 @@ public function get($filename, $scope)
     {
         // First pick up the root level test suite dir
         $paths = glob(
-            FilePathFormatter::format(TESTS_BP) . self::ROOT_SUITE_DIR
+            FilePathFormatter::format($this->getRootSuiteDirPath()) . self::ROOT_SUITE_DIR
```

Comment:
The logic in getRootSuiteDirPath() method is not straight forward. Can we check
MAGENTO_BP . DIRECTORY_SEPARATOR . 'dev/tests/acceptance' and TEST_BP, If they are different, read both paths, otherwise, just read TEST_BP.

Reply:
changed logic as adobe, tested for all suite locations + unit tests and it works. Thanks